xml/System.Security.Cryptography/ECDiffieHellman.xml

```diff
@@ -666,9 +668,10 @@
         <Parameter Name="xmlString" Type="System.String" Index="0" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="xmlString">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="xmlString">The XML string to use to reconstruct the <see cref="T:System.Security.Cryptography.AsymmetricAlgorithm" /> object.</param>
+        <summary>This method throws in all cases.</summary>
         <remarks>To be added.</remarks>
+        <exception cref="System.NotImplementedException">In all cases.</exception>
```

Comment:
```suggestion
        <exception cref="System.NotImplementedException">This exception is always thrown.</exception>
```

Reply:
For the `<exception>` tag "In all cases" is what the style guide says: https://github.com/dotnet/dotnet-api-docs/wiki/Exceptions (last bullet)

Reply:
Ok, I couldn't find that, thanks 😄